src/services/identity/index.ts

```diff
@@ -27,7 +27,7 @@ const NODE_ENV = config.get("env")
 const OTP_SECRET = config.get("auth.otpSecret")
 const OTP_EXPIRY = config.get("auth.otpExpiry")
 
-const IS_LOCAL_DEV = NODE_ENV === "LOCAL_DEV"
+const IS_LOCAL_DEV = NODE_ENV === "dev"
```

Comment:
should we just call it IS_DEV then?

Reply:
Sure, changed in https://github.com/isomerpages/isomercms-backend/pull/659/commits/7989ed4398f329f9d05a56a91cada7b54ee7004a